examples/tree_2d_dgsem/elixir_euler_kelvin_helmholtz_instability_ethz.jl

```diff
@@ -0,0 +1,108 @@
+using OrdinaryDiffEq
```

Comment:
Is "ETH Zurich Kelvin-Helmholtz setup" the canonical name for this? Given that Eitan Tadmor is not from Zurich, I would probably pick a more inconspicuous name such as `_fjordholm_etal` (both for the file name and the initial condition name)

Reply:
Good idea :1:

Reply:
Good idea, I changed it accordingly.